Products/CMFPlone/controlpanel/browser/overview.py

```diff
@@ -45,6 +48,31 @@ def core_versions(self):
     def pil(self):
         return 'PIL' in self.core_versions()
 
+    def server_info(self):
+        wsgi = isinstance(self.request, WSGIRequest)
+        server_name = 'unknown'
+        server_version = ''
+
+        # check for ZServer
+        servers = getattr(getConfiguration(), 'servers', None)
+        if servers and 'ZServer' in servers[0].__module__:
```

Comment:
ZServer also sets SERVER_SOFTWARE , so it could be handled in the same way as the WSGI servers. This would simplify the code.

Reply:
Oh, I missed that. I update the code.